### DISCUSSION TITLE
Issue with Symbol Binding and Compatibility on macOS

### DISCUSSION THREAD

**Discussion by marcoscalderon7** [upscayl/upscayl#637] [Q&A] [answered]
Hi there,

I hope this message finds you well. I'm currently experiencing an issue with the Upscayl application on macOS, and I wanted to bring it to your attention for further investigation.

The error message I'm encountering is related to symbol binding and references to functions in the C++ standard library (libc++). It seems there might be a mismatch between the version of the library used during compilation and the one available on my macOS system.

The application is reporting the following error:

**dyld: lazy symbol binding failed: Symbol not found: __ZNKSt3__115basic_stringbufIcNS_11char_traitsIcEENS_9allocatorIcEEE3strEv
Referenced from: /Applications/Upscayl.app/Contents/Resources/bin/upscayl-bin (which was built for Mac OS X 12.6)
Expected in: /usr/lib/libc++.1.dylib**

I have already checked system requirements, updated the application, and ensured my GPU drivers are up to date. However, the issue persists.

If possible, could you please provide guidance on how to address this symbol binding error or any updates that might be in progress to resolve compatibility issues on macOS?

Thank you for your time and assistance.

Best regards,

**Comment by aaronliu0130** [comment, marked as answer]
Are you sure you’ve checked system requirements, namely that we require macOS 12?

**Comment by marcoscalderon7** [comment]
I apologize, it seems I didn't check the system requirements, including the need for macOS 12, thanks
